x-pack/libbeat/management/generate.go

```diff
@@ -65,13 +65,43 @@ func (r *TransformRegister) Transform(
 // Public config transformation
 // ===========
 
+// handleSimpleConfig perform the necessary checks and transformations in the raw
+// config for the simple configuration case (no `streams` key in the config).
+//
+// Currently only the the `filestream` input is supported for the simple config.
+func handleSimpleConfig(raw *proto.UnitExpectedConfig) (map[string]any, error) {
+	m := raw.Source.AsMap()
+	kind, ok := m["type"].(string)
```

Comment:
this will panic if m["type"] is nil or doesn't exist

Reply:
It does panic https://go.dev/play/p/kxF_OkPTcRO

Reply:
oops, you are right, assert with ok here, was mistake in my test code